content/quickstart_ocaml_project_dune.md

```diff
@@ -166,8 +168,9 @@ $ dune exec ./main.exe
 Hello, world!
 ```
 
-Great! The program is working correctly. Note that since Dune 1.0+beta18, exec command also
-implies build, so going forward we will mostly only use exec to build and run our program.
+Great! The program is working correctly.
+
+**IMPORTANT**: Note that since Dune 1.0+beta18, **exec command also implies build**, so going forward we will mostly only use exec to build and run our program.
```

Comment:
I think reformatting the line to be wider isn't a good idea. It's also not really necessary to mention the pre-release, I think saying that in Dune, exec will build if the executable doesn't exist yet.

Reply:
Done